sbin/unit-tests

```diff
@@ -66,11 +66,6 @@ OSNICK=$($READIES/bin/platform --osnick)
 
 export EXT_TEST_PATH=${BINROOT}/search/example_extension/libexample_extension.so
 
-SKIP_CPP_TESTS=""
-if [[ $ARCH == arm64v8 ]]; then
-	SKIP_CPP_TESTS="IndexTest.testHybridVector"
-fi
```

Comment:
Let's keep the `SKIP_CPP_TESTS=`  expression (without the quotes), so we can add those in the future.